dev/e2e_app/integration_test/permissions/permissions_location_test.dart

```diff
@@ -64,6 +64,10 @@ void main() {
         await $.native.grantPermissionOnlyThisTime();
       }
       await $.pump();
+      await $.pump(Duration(seconds: 5));
+      await $.pump(Duration(seconds: 5));
+      await $.pump();
+      await $.pump();
 
       await tapOkIfGoogleDialogAppears($);
```

Comment:
It could be moved to `tapOkIfGoogleDialogAppears()` and use only one pump with longer duration if really needed